docs/install/network_options.md

```diff
@@ -58,3 +58,92 @@ cp rke2-cilium.yml /var/lib/rancher/rke2/server/manifests/
+
+Multus should be deployed after the cluster is already up and running and has a CNI plugin taking care of the cluster networking, e.g. Canal. After installing Multus, the current CNI plugin becomes the main CNI plugin under Multus transparently and thus, the cluster can be operated as before, as all pods will get an interface from the main plugin automatically. For more information refer to the [multus-cni](https://github.com/k8snetworkplumbingwg/multus-cni/tree/master/docs)
+
+To deploy Multus, use following config:
+
+```
+apiVersion: helm.cattle.io/v1
+kind: HelmChart
+metadata:
+  name: rke2-multus
+  namespace: kube-system
+spec:
+  repo: https://rke2-charts.rancher.io
+  chart: rke2-multus
+```
+
+The config needs to be copied over to the manifests directory:
+
+```
+mkdir -p /var/lib/rancher/rke2/server/manifests/
+cp rke2-multus.yml /var/lib/rancher/rke2/server/manifests/
+```
+
+For more information about the options of the Multus config please refer to the [rke2-charts](https://github.com/rancher/rke2-charts/blob/main-source/packages/rke2-multus/charts/values.yaml)
+
+
+## Using Multus with the containernetworking plugins
+
+To take advantage of Multus, extra CNI plugins should be used to manage the additional network interfaces attached per pod. That is why, when installing the Multus helm chart, the common CNI plugins maintained by the containernetworking team are deployed (e.g. macvlan CNI). For more information about these plugins, check the [containernetworking plugins](https://www.cni.dev/plugins/current).
+
+As explained in the Multus documentation, to use one of these plugins, a NetworkAttachmentDefinition object would need to be created first pointing to one of these CNI plugins. After that, in the pod annotations, we can refer to it so that Multus can start the process to provide one extra interface to that pod. Please follow this example that configures macvlan cni plugin with multus [example](https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/docs/quickstart.md#storing-a-configuration-as-a-custom-resource)
+
+## Using Multus with SRIOV
+
+Using the SRIOV-cni as part of Multus, can help with dataplane acceleration use cases, providing an extra interface in the pod that can achieve very high throughput. SRIOV will not work in all environments and there are several requirements that must be fulfilled to consider the node as sriov capable:
+
+* Physical NIC must support SRIOV (e.g. by checking /sys/class/net/$NIC/device/sriov_totalvfs)
+* The host operating system must activate IOMMU virtualization
+* The vfio driver must be available in the host operating system
```

Comment:
IOMMU & vfio would only be needed for DPDK, right? Should we clarify?

Reply:
I think IOMMU is needed for SRIOV. vfio is not, you are right, there might be alternatives